### ISSUE TITLE
sentient ID claim checker

### ISSUE THREAD

**Issue by augustfr** [tomasg88/mirage-gallery#82]
On the sentient claim page, we need to be able to check if sentient IDs have claimed their tokens or not. This can be seen in the current implementation here: https://account.miragegallery.ai/sentient-claim.

All of the code to do this is in the github. Also this can be tested yourself at the link above with no membership. 

The idea is for people without memberships to check if a membership has or has not claimed artworks.

**Comment by tomasg88** [comment]
# Description
In /membership page in Sentient Tab, we have to add a **Check Membership** button and add the functionality to the current Claim button.

## Tasks

- [ ] Dropdown needs to show 2 projects, Curated and Dreamers.
- [ ] 2nd Dropdown should appear with all the Drops _if Curated is selected in the 1st Dropdown_.
- [ ] Add a _Check Membership_  button that will list all the belongings of that selected membership. That way, users know what would be a fair price for that membership.
- [ ] If Curated is selected and a drop is selected, then the form should show Membership ID input and allow claiming for that specific Drop.
- [ ] If Dreamers is selected in the 1st dropdown, then the form should show Membership ID input and allow claiming for Dreamers

## TL;DR
Memberships can be sold as well. What users do, is check what pieces that membership has claimed in order to understand and calculate how much that membership would cost.
